### DISCUSSION TITLE
printable certificate of completion per section after x% answered correctly

### DISCUSSION THREAD

**Discussion by kingpinzs** [freeCodeCamp/Developer_Quiz_Site#255] [Ideas]
I think it would be a good idea to have a printable/share able certificate after answering x% of questions in a certain section.
This would hinge on the face that each quiz type could be tracked.

**Comment by jdwilkin4** [comment]
I am more interested in the sharable aspect of this than the printable certificate part.
We could add a Tweet button with messages like "I just scored a perfect score on the developerquiz.org site". Or something to that effect. 

As for the printable certificate, I always associate that more so with actual courses than quizzes.
But I would be happy to tag some other freeCodeCamp team members to hear their thoughts. I will tag a few of the team members that were apart of the early stages of the site if they are interested in leaving their thoughts 👍 

@naomi-lgbt @estefaniacn @deniselemonaki

**Reply by naomi-lgbt** [reply]
I like the idea of socialising the gamification - and agree with Jessica about a Tweet button. Maybe `I just scored X% on the <quiz name> - try it yourself? <link>`

A certification here wouldn't really hold any value - the answers are all open source.

**Reply by estefaniacn** [reply]
I agree. I think it's a great idea to add a Tweet button to share the score on Twitter. 👍

**Reply by Verifiedthug** [reply]
Great I will like to participate in this quiz also

**Comment by jdwilkin4** [comment]
@kingpinzs 

Let's go with the tweet button located on the results page.
And we will go with Naomi's suggestion for the wording. 

> I just scored X% on the <quiz name> - try it yourself? <link>

You are free to create an issue with details on this new feature. 👍

**Reply by a2937** [reply]
While no issue has been formally made, I think I made something that looks nice?

**Reply by a2937** [reply]
I can always add/use the css to make it look like a proper button.

**Reply by Verifiedthug** [reply]
Well done boss can you put me through that also

**Reply by jdwilkin4** [reply]
Hi @Verifiedthug ! 

This feature has already been added to the site.
If you are interested in contributing, please read through the [open issues](https://github.com/freeCodeCamp/Developer_Quiz_Site/issues) that currently don't have anyone assigned to them.

**Comment by jdwilkin4** [comment]
Locking this conversation since it has already been implemented.
